server/lib/payments.js

```diff
@@ -194,7 +194,7 @@ export const buildRefundForTransaction = (t, user, data, refundedPaymentProcesso
     // We're handling payment processor fees and host fees in separate transactions
     refund.hostFeeInHostCurrency = 0;
     refund.paymentProcessorFeeInHostCurrency = 0;
-    refund.netAmountInCollectiveCurrency = -netAmount({ ...t, paymentProcessorFeeInHostCurrency: 0 });
+    refund.netAmountInCollectiveCurrency = netAmount(refund);
```

Comment:
This line was wrong: `{ ...t }` would not end up as intended, because values are actually stored in `...t.dataValues`. As a result, we ended up with `NaN` stored in `netAmountInCollectiveCurrency` which resulted in having it automatically re-computed later, but with some inconsistencies.